### ISSUE TITLE
Deploy Nuclei in Kubernetes + Helm Chart

### ISSUE THREAD

**Issue by heywoodlh** [projectdiscovery/nuclei#2452]
### Please describe your feature request:
There is no documentation, Helm chart, etc. that I could find on deploying Nuclei in Kubernetes. As a result, I have created a functional Helm chart for Nuclei here: https://github.com/heywoodlh/argo-apps/tree/master/charts/nuclei

My Helm chart runs the Nuclei container will run as a cron job in Kubernetes on a recurring/scheduled basis. It also has a `values.yaml` that has variables for configuring your targets as well as everything else in Nuclei's config.

### Describe the use case of this feature:
Be able to deploy Nuclei in a K8s cluster.

### Action Needed:
I'm mostly just curious if this is a desired feature -- I could potentially write a PR and submit the changes there.

**Comment by ehsandeep** [comment]
Hi @heywoodlh,

Thank you for creating this issue. This is something we haven't tried ourselves, and others may be interested in the same thing. If you can add instructions or information about deploying Nuclei in a K8s cluster, that would be very helpful.

**Comment by heywoodlh** [comment]
@ehsandeep here is the PR: https://github.com/projectdiscovery/nuclei/pull/2557

Some things I'd like to note:

- Using the Helm chart as its base, I was able to write a Helm chart at my place of employment that automatically scans all our web applications in our cluster. I think Nuclei's flexibility puts it in a unique position to fill the need of network-level vulnerability scanning _and_ fit very nicely into Kubernetes. I have found that most Kubernetes-focused vulnerability scanners only focus on the OS and package level within containers. Nuclei could very much fill that gap in a lot of Kubernetes deployments, imo.
- The Helm chart at the moment has no method of persistence for Nuclei templates. So the Helm chart is wholly dependent upon the community templates. I'll probably update the chart with this functionality sometime this week.
- Something like [Mend Renovate](https://www.mend.io/free-developer-tools/renovate/) should be implemented to automatically keep the container image tags up to date in the `values.yaml` file
